com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/adapter/handler/ConfigurationDoneRequestHandler.java

```diff
@@ -63,7 +68,20 @@ private void handleDebugEvent(DebugEvent debugEvent, IDebugSession debugSession,
         Event event = debugEvent.event;
         boolean isImportantEvent = true;
         if (event instanceof VMStartEvent) {
-            // do nothing.
+            if (context.isVmStopOnEntry()) {
+                EventRequestManager manager = debugSession.getVM().eventRequestManager();
+                ArrayList<MethodEntryRequest> legacy = new ArrayList<>(manager.methodEntryRequests());
```

Comment:
Each debug session has only one VMStartEvent, it seems deleting legacy request is meaningless.